app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -297,6 +300,26 @@ def log_idv_verification_submitted_event(success: false)
       )
     end
 
+    def log_irs_threatmetrix_fraud_check_event(result)
```

Comment:
Should we take "irs" out of the name here since users from all partners go through TMX?

Reply:
Possibly, I'm trying to figure out what a `FraudReviewRequest` is and if that is something we need to keep creating in this method

Reply:
fraud review request is the process where fraud team investigates an account, and the identity verification is not approved until after that has been adjudicated

Reply:
I am going to split this method into two: one to create the `FraudReviewRequest` and one to do the attempts API dance (on the assumption that the attempts API stuff will get cleaned up later)